### PR TITLE
Fix reaction adds to RFR messages if reacting with a unicode emoji

### LINKED ISSUE #245: [RFR] RFR Message breaks when reacting with a Unicode emoji after setup
### Describe the bug
After adding a Unicode emoji, RFR message will not add/remove roles correctly once someone reacts with a Unicode emoji, then removes all emoji options one by one. Does not get fixed by `fixEmbed`

### To Reproduce
Numbered steps to reproduce the behaviour:
1. `k!rfr create` - Create an RFR Message
2. `k!rfr edit addRoles`, then add an `<emoji>,<role>` combination with at least one Unicode emoji, e.g. 3️⃣
3. React to message after setup with a Unicode Emoji

### Expected behaviour
Add correct role associated with emoji to person who reacted, don't remove emoji, role and reaction from RFR message after react.

### Screenshots
image

### Additional context
Add any other context about the problem here, as well as the version of KoalaBot this affects.
Seen on v0.4.3

### PR DESCRIPTION
**Pull request by Unknownlocal**
## Summary

`close #245`
Fixes bug where RFR messages break when having a unicode emoji
## Checklist

- [x] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

<br>

- [x] Have you tested the changes? ([pytest](https://docs.pytest.org/) & [dpytest](https://dpytest.readthedocs.io/))
- [x] Have you followed [PEP-8](https://www.python.org/dev/peps/pep-0008/) for naming and styling?  
- [x] Has your code been properly documented with RestructuredText docstrings?
- [x] Have you added your changes to `CHANGELOG.md` under the `[Unreleased]` heading?
- [x] If your code added new bot commands, have you updated `documentation.json`?

<br>

- [x] All of this code is your own, or follows the author repo's licence.

### CODE REVIEW

**Review by Kaspiaan — COMMENTED** [review]
Everything looks good, I saw an issue with a version of MacOS so am refraining from approving as I can't properly look at the files right now

**Comment by JayDwee** [comment]
Coverage needs increasing, then we can merge @Unknownlocal

**Comment by Unknownlocal** [comment]
@JayDwee This good now?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,29 @@
 # Changelog
-All notable changes to KoalaBot will be documented in this file.
-A lot of these commands will only be available to administrators
+
+All notable changes to KoalaBot will be documented in this file. A lot of these commands will only be available to
+administrators
```
Comment (JayDwee) [inline CHANGELOG.md:4]: Not a useful change, please undo

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
+
 - Testing updated to use builders in dpytest 0.5.0
 - Additional option with `--config <path to config>` to choose where databases are stored
 
 ## [0.4.3] - 14-05-2021
+
 ### Announce
+
 - Announce is hidden in help when not enabled
+
```
Comment (JayDwee) [inline CHANGELOG.md:26]: extra spaces aren't required

File: cogs/ReactForRole.py [hunk 3]
```diff
@@ -763,6 +765,8 @@ async def on_raw_reaction_remove(self, payload: discord.RawReactionActionEvent):
                                                           payload.message_id, payload.user_id)
             if not member_role or member_role[0].bot:
                 return
+            await self.bot.get_user(payload.user_id).send(
+                f"trying to remove \\{member_role[1]} from your roles because you reacted with \\{payload.emoji}")
```
Comment (JayDwee) [inline cogs/ReactForRole.py:769]: Is this required? (feel free to justify, I'm just wondering)

File: cogs/ReactForRole.py [hunk 4]
```diff
@@ -845,6 +852,7 @@ async def get_role_member_info(self, emoji_reacted: discord.PartialEmoji, guild_
                 return
             role_str = field
             role: discord.Role = discord.utils.get(guild.roles, mention=role_str.lstrip().rstrip())
+            print(f"emoji identified {emoji_reacted} and linked to {role}")
```
Comment (JayDwee) [inline cogs/ReactForRole.py:855]: Prints need removing, or changing to logging

**Review by JayDwee — CHANGES_REQUESTED** [review]
just a couple changes, but looks good other than that

File: CHANGELOG.md (hunk 1 above)
Comment (Unknownlocal) [reply CHANGELOG.md:4]: I think it auto-formatted everything before I pushed, will undo

**Review by JayDwee — APPROVED** [review]
👍

**Pull request merged**